index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <title>GPU-friendly Stroke Expansion</title>
+  <meta charset="UTF-8">
+  <meta property="og:description" content="GPU-friendly Stroke Expansion"/>
+  <link href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">
+  <style>
+    body {
+      font-family: 'Lato', Verdana, Helvetica, sans-serif;
```

Comment:
The GF call to bring in Lato is missing, so this renders as Verdana on my system.

Reply:
Done.